modules/core/core-project/src/main/java/com/enonic/xp/core/impl/project/ProjectEvents.java

```diff
@@ -0,0 +1,25 @@
+package com.enonic.xp.core.impl.project;
+
+import com.enonic.xp.event.Event;
+import com.enonic.xp.project.ProjectName;
+
+public class ProjectEvents
+{
+    public static final String PROJECT_NAME_KEY = "name";
+
+    public final static String CREATED_EVENT_TYPE = "project.created";
```

Comment:
Codacy found an issue: ['static' modifier out of order with the JLS suggestions.](https://app.codacy.com/manual/enonic/xp/pullRequest?prid=6409740)